Disable the output invocation in jit

eellison opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #71173
* __->__ #71138
* #70656
* #70464
* #71144

Differential Revision: [D33521059](https://our.internmc.facebook.com/intern/diff/D33521059)

Commit:
Disable the output invocation in jit

[ghstack-poisoned]

Commit:
Update on "Disable the output invocation in jit"




[ghstack-poisoned]

navahgar commented on torch/csrc/jit/passes/tensorexpr_fuser.cpp:
```diff
@@ -1258,6 +1258,10 @@ Operation createTensorExprOp(const Node* node) {
   if (node->hasAttribute(attr::symbolic_shape_inputs)) {
     sym_shapes = node->is(attr::symbolic_shape_inputs);
   }
+  bool allow_stack_outputs = false;
```
This bool should be used below before calling `runWithAllocatedOutputs`.

eellison replied:
thanks yea - I didn't mean to push this commit out yet... still have to add test

Commit:
Update on "Disable the output invocation in jit"




[ghstack-poisoned]

eellison commented:
@eellison has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33521059).

Commit:
Update on "Disable the output invocation in jit"


Differential Revision: [D33521059](https://our.internmc.facebook.com/intern/diff/D33521059)

[ghstack-poisoned]

Commit:
Update on "Disable the output invocation in jit"


Differential Revision: [D33521059](https://our.internmc.facebook.com/intern/diff/D33521059)

[ghstack-poisoned]

eellison commented:
@eellison has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33521059).

navahgar reviewed (approved):
Thanks for adding this.

Commit:
Update on "Disable the output invocation in jit"


Differential Revision: [D33521059](https://our.internmc.facebook.com/intern/diff/D33521059)

[ghstack-poisoned]

eellison commented:
@eellison has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33521059).

Commit:
Update on "Disable the output invocation in jit"


Differential Revision: [D33521059](https://our.internmc.facebook.com/intern/diff/D33521059)

[ghstack-poisoned]

eellison commented:
@eellison has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33521059).

Commit:
Update on "Disable the output invocation in jit"


Differential Revision: [D33521059](https://our.internmc.facebook.com/intern/diff/D33521059)

[ghstack-poisoned]

eellison commented:
@eellison has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33521059).
